[Doc] correct ceph migration steps (#757)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc

$ sudo cephadm shell -- ceph mgr stat
----

> . Fail the `ceph-mgr` if it is active on the source :

Reviewer: ```suggestion
. Fail the `ceph-mgr` if it is active on the source:
```

Author: done

> . Fail the `ceph-mgr` if it is active on the source or draining node :

Reviewer 2: can we simply remove `or target node` here? I wouldn't add `or draining node`

Author: done

+
----
$ for label in mon mgr _admin; do
    sudo cephadm shell -- ceph orch host label rm <source_node> $label;
done
----
+
* Replace `<source_node>` with the hostname of the source node.

> . (Optional) Ensure to remove Ceph Monitor daemon from the source node if it is still running :

Reviewer: ```suggestion
. Optional: Ensure that you remove the Ceph Monitor daemon from the source node if it is still running:
```

Author: done

> . (Optional) Ensure to remove Ceph Monitor daemon from the source node if it is still running ):

Reviewer 2: Remove `)` at the end

Author: done

> . (Optional) Ensure that you remove the Ceph Monitor daemon from the source node if it is still running:

Reviewer: ```suggestion
. Optional: Ensure that you remove the Ceph Monitor daemon from the source node if it is still running:
```
Our style for optional steps is "Optional:", not "(Optional)."

Author: done
